docusaurus/docs/get-started/get-started.mdx

```diff
@@ -243,13 +243,20 @@ mage -v build:linux
 | Option         | Description                                  | Example               |
 | -------------- | -------------------------------------------- | --------------------- |
 | `build:[arch]` | Builds a binary for a specific architecture. | `mage -v build:Linux` |
+| `build:backend` | Builds a production build for the current platform. | `mage -v build:backend` |
 
 List all available Mage targets for additional commands:
 
 ```bash
 mage -l
 ```
 
+:::note
+
+There is a mismatch between AMD, ARM64, and the Docker container which may cause the backend Delve debugger dev env to fail to run on Apple silicon.
```

Comment:
@oshirohugo can you confirm this is still the case?

Reply:
This was improved, but it seems it still not perfect and it will only work in Apple silicon after two runs.
@jackw you worked on that, can you confirm?

Reply:
@oshirohugo I'll do some testing here and get back to you. It was a while since I tried this debugger.